### PR TITLE
Added Kannada News Classification Dataset

### PR DESCRIPTION
**Pull request by Akash190104**
Added Kannada News Classification Dataset. 
## Checklist for adding MMTEB dataset

Reason for dataset addition:

This is the first classification task for Kannada.

- [x] I have tested that the dataset runs with the `mteb` package.
- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb run -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] I have considered the size of the dataset and reduced it if it is too big (2048 examples is typically large enough for most tasks)
- [x] I have filled out the metadata object in the dataset file (find documentation on it [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#2-creating-the-metadata-object)).
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 
- [x] I have added points for my submission to the [points folder](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) using the PR number as the filename (e.g. `438.jsonl`).

### CODE REVIEW

File: mteb/tasks/Classification/kan/Kannada_News_Classification.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        form=["written"],
+        domains=["News"],
+        task_subtypes=["Topic classification"],
+        license=None,
+        socioeconomic_status=None,
+        annotations_creators=None,
+        dialect=None,
+        text_creation="found",
+        bibtex_citation=None,
+        n_samples={"train": 6460},
```
Comment (isaac-chung) [inline mteb/tasks/Classification/kan/Kannada_News_Classification.py:32]: Could define a var N_SAMPLES=2048 and use it below as well.

File: mteb/tasks/Classification/kan/Kannada_News_Classification.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        eval_langs=["kan-Knda"],
+        main_score="accuracy",
+        date=None,
+        form=["written"],
+        domains=["News"],
+        task_subtypes=["Topic classification"],
+        license=None,
+        socioeconomic_status=None,
+        annotations_creators=None,
+        dialect=None,
```
Comment (isaac-chung) [inline mteb/tasks/Classification/kan/Kannada_News_Classification.py:29]:
```suggestion
        dialect=[],
```

File: mteb/tasks/Classification/kan/Kannada_News_Classification.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        eval_splits=["train"],
+        eval_langs=["kan-Knda"],
+        main_score="accuracy",
+        date=None,
+        form=["written"],
+        domains=["News"],
+        task_subtypes=["Topic classification"],
+        license=None,
+        socioeconomic_status=None,
+        annotations_creators=None,
```
Comment (isaac-chung) [inline mteb/tasks/Classification/kan/Kannada_News_Classification.py:28]: Could you please fill these out based on https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#2-creating-the-metadata-object?

**Review by isaac-chung — CHANGES_REQUESTED** [review]
Thanks for adding this! Just a few comments.
Feel free to add 2 points for the dataset and 2 points to me for the review.

**Comment by Akash190104** [comment]
@isaac-chung Thank you for your suggestions. I would make the changes, ina few hours. Just to clarify, based on this, https://github.com/embeddings-benchmark/mteb/tree/main/docs/mmteb am I eligible for the bonus here?
It's okay either way. Just wanted to understand the system. Thanks!

**Comment by isaac-chung** [comment]
Yep, you will be. Follow the last point on the PR checklist to add a file to the points folder.

**Comment by Akash190104** [comment]
Made changes.

**Review by isaac-chung — APPROVED** [review]
Thanks for adding this! FYI kan is covered by `IndicSentimentClassification`, so this dataset would yield 2 points instead of 6.

**Pull request merged**